### DISCUSSION TITLE
How to Access Forking Renovate Logs?

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#18618] [Request Help] [answered]
[The dashboard](https://app.renovatebot.com/dashboard) seems super useful for debugging, but it appears only to be available for the sister app, Mend Renovate. How do people debug issues with Forking Renovate?

**Comment by rarkins** [comment]
Forking Renovate logs are not currently available. Only a small fraction of total people use it compared to the main one.

**Reply by Kurt-von-Laven** [reply, marked as answer]
https://developer.mend.io now supports Forking Renovate. Bravo!
